packages/evm/src/providers/viem.ts

```diff
@@ -67,3 +67,61 @@ export const ARCTIC_1_VIEM_CHAIN: Chain = {
 	sourceId: SeiChainInfo.devnet.chainId,
 	testnet: true
 };
+
+/**
+ * Creates and returns a Viem Chain with the default atlantic-2 configs and precompile contracts added.
+ * @example
+ * React: Use with `createWagmiConfig` for use in a WagmiProvider:
+ * ```tsx
+ * import { ReactNode } from 'react';
+ * import { WagmiProvider } from 'wagmi';
+ * import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+ * import { ATLANTIC_2_VIEM_CHAIN, createWagmiConfig } from '@sei-js/evm';
+ *
+ * const queryClient = new QueryClient();
+ *
+ * export const WalletProvider = ({ children }: { children: ReactNode }) => {
+ *  return (
+ *    <WagmiProvider config={createWagmiConfig(ATLANTIC_2_VIEM_CHAIN)}>
+ *      <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
+ *    </WagmiProvider>
+ *  );
+ * };
+ * ```
+ * @returns A 'viem' package chain compatible with Wagmi and Viem.
+ * @category Viem
+ */
+export const ATLANTIC_2_VIEM_CHAIN: Chain = {
+	blockExplorers: {
+		default: {
+			name: 'SeiTrace',
+			url: 'https://seitrace.com'
+		}
+	},
+	// Precompiles are at the same address.
+	contracts: {
+		addressPrecompile: { address: ARCTIC_1_ADDRESS_PRECOMPILE_ADDRESS },
```

Comment:
These should be updated to use Atlantic-2 precompile addresses unless all chains use the same address. Then we can just refactor all chains to be just ‘ADDRESS_PRECOMPILE_ADDRESS’

Reply:
Ya actually I think they likely should be the same?
https://github.com/sei-protocol/sei-chain/blob/seiv2/precompiles/staking/Staking.sol
Given that the addresses are written into the code itself

Reply:
Just updated so they all just say <>_address_precompile

Reply:
Psu confirmed that addresses are same throughout